src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetDateCommand.cs

```diff
@@ -217,6 +217,19 @@ public int Millisecond
         [ArgumentCompletions("FileDate", "FileDateUniversal", "FileDateTime", "FileDateTimeUniversal")]
         public string Format { get; set; }
 
+        /// <summary>
+        /// Convert date to UTC before formatting
+        /// </summary>
+        [Parameter(ParameterSetName = "net")]
+        public SwitchParameter AsUTC
+        {
+            get { return _asUTC; }
+
+            set { _asUTC = value; }
+        }
+
+        private SwitchParameter _asUTC = false;
```

Comment:
Please don't use private fields.
`public SwitchParameter AsUTC { get; set; }`

Reply:
done.
(fwiw, I was trying to replicate how other fields in this Cmdlet are handled with public/private fields)

Reply:
fwiw, the Codacy PR tool disagrees with you :)

https://app.codacy.com/manual/TravisEz13/PowerShell_PowerShell/pullRequest?prid=4837266